packages/block/test/from-rpc.spec.ts

```diff
@@ -25,4 +28,31 @@ tape('[fromRPC]: block #2924874', function (t) {
     st.ok(block.validateUnclesHash())
     st.end()
   })
+
+  t.test(
+    'should create a block with json data that includes a difficulty parameter of type hex string',
+    function (st) {
+      const blockDifficultyAsHex = blockFromRpc(blockDataDifficultyAsHex)
+      st.ok(blockDifficultyAsHex)
```

Comment:
I think in these three tests you added you should check that the parameter is actually the result you expected (e.g. `st.ok(blockDifficultyAsHex.header.difficulty.eqn(1))`